cunit/aaa-db.testc

```diff
@@ -190,15 +190,6 @@ static char *make_basedir(const char * const *reldirs)
 
 static int skiptest()
 {
-    /* cunit.pl doesn't play nice with #ifdef'ed CUNIT_PARAMS */
-
-    if (!strcmp(backend, "zeroskip")) {
-#ifdef HAVE_ZEROSKIP
-        return 0;
-#else
-        return 1;
-#endif
-    }
```

Comment:
I think you could remove the whole `skiptest()` function and all calls to it, it only existed because of zeroskip